io.openems.edge.batteryinverter.api/src/io/openems/edge/batteryinverter/api/BatteryInverterConstraint.java

```diff
@@ -0,0 +1,29 @@
+package io.openems.edge.batteryinverter.api;
+
+import io.openems.edge.ess.power.api.Phase;
+import io.openems.edge.ess.power.api.Pwr;
+import io.openems.edge.ess.power.api.Relationship;
+
+/**
+ * Wraps a Constraint for a {@link ManagedSymmetricBatteryInverter}.
+ */
+public class BatteryInverterConstraint {
+
+	public final String description;
+	public final Phase phase;
+	public final Pwr pwr;
+	public final Relationship relationship;
+	public final double value;
+
+	public static BatteryInverterConstraint[] NO_CONSTRAINTS = new BatteryInverterConstraint[] {};
+
+	public BatteryInverterConstraint(String description, Phase phase, Pwr pwr, Relationship relationship,
+			double value) {
+		this.description = description;
+		this.phase = phase;
+		this.pwr = pwr;
+		this.relationship = relationship;
+		this.value = value;
+	}
+
+}
```

Comment:
Unfortunately not, because `io.openems.edge.ess.power.api.Constraint` is built on 'io.openems.edge.ess.power.api.LinearCoefficient` which uses `io.openems.edge.ess.power.api.Coefficient` - and that one referrs directly to an Ess-ID, which I don't have in the Battery-Inverter.

I did not want to mess with the existing structure and the Ess-Power solver, so I introduced a special, simpler variant of the Constraints for Battery-Inverter.